src/plugins/maps_legacy/public/map/service_settings.test.js

```diff
@@ -42,11 +42,18 @@ import EMS_STYLE_ROAD_MAP_BRIGHT from '../__tests__/map/ems_mocks/sample_style_b
 import EMS_STYLE_ROAD_MAP_DESATURATED from '../__tests__/map/ems_mocks/sample_style_desaturated';
 import EMS_STYLE_DARK_MAP from '../__tests__/map/ems_mocks/sample_style_dark';
 import { ORIGIN } from '../common/constants/origin';
-import { ServiceSettings } from './service_settings';
+import { ServiceSettings, DEFAULT_SERVICE } from './service_settings';
+
+function assertObject(actual, expected) {
```

Comment:
Can we use `.toMatchObject()` instead? https://jestjs.io/docs/expect#tomatchobjectobject

Reply:
sure, will update